### DISCUSSION TITLE
Is it possible to read the Arabic alphabet ?

### DISCUSSION THREAD

**Discussion by luizefalves** [py-pdf/pypdf#687] [Q&A]
Hello, I was trying to read a PDF with some Arabic texts for a client.
After reading the file, just the English parts where properly converted,  the Arabic text returned a sad ASCII like "b'\xc3\x89\xc2\xa7Z\nd\....". 
Any tips on how to solve this ?
Thank you so much.

**Comment by MartinThoma** [comment]
#464 might help

Do you have a simple / short example file + the expected output?
